dali/pipeline/operator/op_schema.h

```diff
@@ -82,10 +82,20 @@ class DLL_PUBLIC OpSchema {
   DLL_PUBLIC inline ~OpSchema() = default;
 
   /**
-   * @brief Returns the name of this operator.
+   * @brief Returns the schema name of this operator.
    */
   DLL_PUBLIC const std::string &name() const;
 
+  /**
+   * @brief Returns the module path of this operator.
+   */
+  DLL_PUBLIC const std::vector<std::string> &ModulePath() const;
+
+  /**
+   * @brief Returns the camel case name of the operator (without the module path)
+   */
+  DLL_PUBLIC const std::string &OperatorName() const;
```

Comment:
How is that different from the schema name? Why do we need both?

Reply:
Returns the camel case name of the operator *(without the module path)*